server/storage/backend/batch_tx.go

```diff
@@ -324,6 +324,15 @@ func (t *batchTxBuffered) unsafeCommit(stop bool) {
 			}
 		}(t.backend.readTx.tx, t.backend.readTx.txWg)
 		t.backend.readTx.reset()
+		version := t.backend.readTx.buf.bufVersion
+		// t.backend.readTx.buf is used for ConcurrentReadTxNoCopyMode
+		// t.backend.readTx.buf can not be reset simply
+		// create a new one, old one can still use
+		t.backend.readTx.buf = &txReadBuffer{
```

Comment:
I see that the line` rt.buf.reset()` is removed in `readTx.reset()`, instead you create a new `txReadBuffer` here for safety. The logic looks good to me, but the comment above isn't clear enough, and other developers may get confused when they read the source code. 

So suggest to reword the above comment to something like below:
```
The buf isn't reset in readTx.reset() because it may still be used by another range 
sessions at the moment. But it's safe to create a new txReadBuffer, because the old 
one keeps unchanged. 
```